pwa/app/routes/_index.tsx

```diff
@@ -12,11 +14,34 @@ export async function loader() {
     });
   }
 
+  const events = await getValidEvents(status.data.current_season);
+
   return json({
     status: status.data,
+    events: events,
   });
 }
 
+export async function getValidEvents(year: number) {
+  const events = await getEventsByYear({ year });
+
+  if (events.status !== 200) {
+    throw new Response(null, {
+      status: 500,
+    });
+  }
+
+  if (events.data.length === 0) {
+    throw new Response(null, {
+      status: 404,
```

Comment:
I dont think we want to 404 or 500 the homepage just because there are no events.

Reply:
I think it should be an issue...good way to encourage an event every week ;)